drivers/sensor/lis2dw12/lis2dw12.h

```diff
@@ -70,6 +70,8 @@ struct lis2dw12_device_config {
 	uint8_t range;
 	uint8_t bw_filt;
 	bool low_noise;
+	bool hp_filter_path;
```

Comment:
It's @avisconti 's driver so this is not a blocking request, but it seems like you could save a little flash by bit-packing these.